### ISSUE TITLE
MXnet suddenly can't connect with CUDA anymore in R

### ISSUE THREAD

**Issue by AgatheBauer** [apache/incubator-mxnet#15682]
I'm using MXnet v1.4 in R3.5.3 with CUDA v9.0 and cudnn v7

I had everything working, but suddenly when I'm calling

`library('mxnet')`

I'm getting this popup error message:

<img width="326" alt="MXNet problem" src="https://user-images.githubusercontent.com/11364795/62011355-c07aab00-b177-11e9-96cb-49be5039ae9b.PNG">

> The procedure entry point cudnnRNNBackwardWeightsEx
> could not be located in the dynamic link library ....RPath\3.5\mxnet\libs\x64\libmxnet.dll.

I swear I didn't touch anything. I tried to fix the issue by reinstalling CUDA and MXnet, without success. Did anybody encounter this issue yet?

This is all my system related info that I'm able to get as a non-professional programmer:

----------Python Info----------
Version      : 3.7.3
Compiler     : MSC v.1915 64 bit (AMD64)
Build        : ('default', 'Mar 27 2019 17:13:21')
Arch         : ('64bit', 'WindowsPE')
------------Pip Info-----------
Version      : 19.0.3
Directory    : C:\Users\simeo\Anaconda3\lib\site-packages\pip
----------MXNet Info-----------
No MXNet installed.
----------System Info----------
Platform     : Windows-10-10.0.18362-SP0
system       : Windows
node         : Simeon-PC
release      : 10
version      : 10.0.18362
----------Hardware Info----------
machine      : AMD64
processor    : Intel64 Family 6 Model 42 Stepping 7, GenuineIntel
Name
Intel(R) Core(TM) i7-2600 CPU @ 3.40GHz

----------Network Test----------
Setting timeout: 10
Timing for MXNet: https://github.com/apache/incubator-mxnet, DNS: 0.0497 sec, LOAD: 1.7985 sec.
Timing for Gluon Tutorial(en): http://gluon.mxnet.io, DNS: 0.0636 sec, LOAD: 1.1705 sec.
Timing for Gluon Tutorial(cn): https://zh.gluon.ai, DNS: 0.1196 sec, LOAD: 0.8515 sec.
Timing for FashionMNIST: https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/dataset/fashion-mnist/train-labels-idx1-ubyte.gz, DNS: 0.0375 sec, LOAD: 1.2377 sec.
Timing for PYPI: https://pypi.python.org/pypi/pip, DNS: 0.0161 sec, LOAD: 0.7398 sec.
Timing for Conda: https://repo.continuum.io/pkgs/free/, DNS: 0.0353 sec, LOAD: 0.4305 sec.

----------R session info----------

R version 3.5.3 (2019-03-11)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18362)

Matrix products: default

locale:
[1] LC_COLLATE=German_Germany.1252  LC_CTYPE=German_Germany.1252    LC_MONETARY=German_Germany.1252
[4] LC_NUMERIC=C                    LC_TIME=German_Germany.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.1             compiler_3.5.3         pillar_1.4.2           RColorBrewer_1.1-2    
 [5] influenceR_0.1.0       viridis_0.5.1          base64enc_0.1-3        keras_2.2.4.1         
 [9] tools_3.5.3            zeallot_0.1.0          digest_0.6.20          viridisLite_0.3.0     
[13] gtable_0.3.0           jsonlite_1.6           tibble_2.1.3           lattice_0.20-38       
[17] rgexf_0.15.3           pkgconfig_2.0.2        rlang_0.4.0            Matrix_1.2-15         
[21] igraph_1.2.4.1         rstudioapi_0.10        gridExtra_2.3          stringr_1.4.0         
[25] downloader_0.4         DiagrammeR_1.0.1       dplyr_0.8.3            generics_0.0.2        
[29] htmlwidgets_1.3        vctrs_0.2.0            hms_0.5.0              grid_3.5.3            
[33] tidyselect_0.2.5       reticulate_1.13.0-9000 glue_1.3.1             R6_2.4.0              
[37] Rook_1.1-1             XML_3.98-1.20          ggplot2_3.2.0          tidyr_0.8.3           
[41] purrr_0.3.2            readr_1.3.1            magrittr_1.5           whisker_0.3-2         
[45] scales_1.0.0           backports_1.1.4        tfruns_1.4             htmltools_0.3.6       
[49] assertthat_0.2.1       colorspace_1.4-1       brew_1.0-6             tensorflow_1.13.1     
[53] stringi_1.4.3          visNetwork_2.0.7       lazyeval_0.2.2         munsell_0.5.0         
[57] crayon_1.3.4

**Comment by AgatheBauer** [comment]
Must be a problem outside of MXnet. Tensorflow doesn't work with GPU acceleration either.
